TASK-2/ChatBot/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿void ChatBotImitation()
+{
+    DateTime dateNow;
+    dateNow = DateTime.Now;
+
+    Console.Write("Hello, enter your lastname ");
+    var lastname = Console.ReadLine();
+    while (!lastname.All(Char.IsLetter) || string.IsNullOrWhiteSpace(lastname))
+    {
+        Console.Write("Please, enter correct lastname ");
+        lastname = Console.ReadLine();
+    }
+
+
+    Console.Write("Hello, enter your name ");
+    var name = Console.ReadLine();
+    while (!name.All(Char.IsLetter) || string.IsNullOrWhiteSpace(name))
+    {
+        Console.Write("Please, enter correct name ");
+        name = Console.ReadLine();
+    }
```

Comment:
It does not fit the task. Need a transformation, not a validation. Read the task
Don't force the user to type a million times his name